packages/react/slider/src/Slider.tsx

```diff
@@ -150,13 +162,19 @@ const Slider = React.forwardRef<SliderElement, SliderProps>(
               data-disabled={disabled ? '' : undefined}
               {...sliderProps}
               ref={composedRefs}
+              onPointerDown={composeEventHandlers(sliderProps.onPointerDown, () => {
+                if (!disabled) valuesBeforeSlideStartRef.current = values;
```

Comment:
Looking at this made me curious about the behaviour of `onSlideStart`, when does that fire and what's the expectation? I would have expected to see this there but seems this wouldn't be accurate?

Reply:
Yeah that's what I originally tried, but the realised there are cases where it isn't called (basically when pointer down on a thumb):

<img width="811" alt="CleanShot 2022-10-10 at 11 46 28@2x" src="https://user-images.githubusercontent.com/1539897/194849234-77285195-72cc-47cc-9483-0d7bb8a462cb.png">

I believe that is so that we don't see a jump straight as we point down, but rather as we start sliding (because of the thumb offset).

Reply:
> I believe that is so that we don't see a jump straight as we point down, but rather as we start sliding (because of the thumb offset).

Ah yes, I can see this now 👍